tuned/gtk/gui_profile_loader.py

```diff
@@ -103,6 +103,11 @@ def _load_all_profiles(self):
 #                         raise managerException.ManagerException("Can not make profile")
 #                         print "can not make \""+ profile +"\" profile without correct config with path: " + d
 
+    def _is_dir_profile(self, path):
+        is_dir = os.path.isdir(path)
```

Comment:
You can (and should, to avoid unnecessary execution) immediatelly return False if it's not a directory.

Reply:
Changed.